### PR TITLE
WIP: django-filer

### PR DESCRIPTION
**Pull request by dimastbk**
Some compatibility with https://github.com/django-cms/django-filer: directory listing, view files, delete comfirmation. In beta version django-filer will broken file_icon in directory listing, after their release need fix this.
image
image

### CODE REVIEW

File: jazzmin/static/jazzmin/css/main.css [hunk 1]
```diff
@@ -829,3 +829,9 @@ body.jazzmin-login-page {
 .callout {
     color: black;
 }
+
+/* tweeks for django-filer*/
+#content-main {
```
Comment (farridav) [inline jazzmin/static/jazzmin/css/main.css:834]: Can this be targeted to django-filer somehow ? are there any classes added to parent elements that will stop this affecting other `#content-main` elements  as its a fairly common id for a container

**Comment by farridav** [comment]
Looking good, if we can fixup that css rule we can get this in :)

**Comment by dimastbk** [comment]
@farridav in jazzmin 2.4.5 compatibility with filer is broken, because filer does not add current_app to the request. Some related issue in filer repo: https://github.com/django-cms/django-filer/issues/318.

File: jazzmin/static/jazzmin/css/main.css (hunk 1 above)
Comment (dimastbk) [reply jazzmin/static/jazzmin/css/main.css]: @farridav I change this to more specific selector `.navigator-top-nav + #content-main`. Child element of content-main has `width: 100%;`.

**Comment by farridav** [comment]
I have just put out a release that _should_ fix the `current_app` issue .. i think this needs a rebase, then we can look to get it in

**Comment by dimastbk** [comment]
@farridav done

**Pull request merged**